### PR TITLE
Split useInvitation into two

### PR DESCRIPTION
**Pull request by ismailToyran**
### Project organization
- Related [trello card](https://trello.com/c/cPl7yMm8/1178-issue-referral-system?filter=member:ismailtoyran)

### Description
Split `useInvitation` into two.
So user can use `useCreateInvitation` with `signer` required and `useAcceptInvitation` without.

### CODE REVIEW

File: example/package.json [hunk 1]
```diff
@@ -9,8 +9,8 @@
     "lint": "next lint && prettier \"**/*.{ts,tsx,json,md}\" --check"
   },
   "dependencies": {
-    "@liteflow/core": "file:../liteflow-core-1.3.0.tgz",
-    "@liteflow/react": "file:../liteflow-react-1.3.0.tgz",
+    "@liteflow/core": "file:../liteflow-core-2.0.0.tgz",
+    "@liteflow/react": "file:../liteflow-react-2.0.0.tgz",
```
Comment (ismailToyran) [inline example/package.json:13]: Build was failing because of the versioning on example folder

File: packages/react/src/useAcceptInvitation.ts [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+  const accept = useCallback(
+    async (invitationId: string) => {
+      try {
+        setAccepting(true)
+        // Ensure that there is no token in the context
+        resetAuthenticationToken()
+        const { acceptInvitation } = await sdk.AcceptInvitation({
+          id: invitationId,
+        })
```
Comment (ismailToyran) [inline packages/react/src/useAcceptInvitation.ts:32]: Can we just use plain graphql query?
Or maybe move this query directly to the starter kit?
There is no need to use sdk here.
Sdk is the issue here.
I've moved this query directly to the starter kit and used normal query and it worked without a problem.
As this is a no log in required process, should be better to have it on login page only?

Comment (antho1404) [reply packages/react/src/useAcceptInvitation.ts:32]: By moving it directly to the starter kit, you benefitted from the fact that the starter kit graphql client doesn't have an issue with the switch of account, while the initialization of the SDK seems to have a problem with it.
Resetting the token is not the solution here as this function should not control the authentication flow, but we need to ensure that when it's used, the actual SDK is reset correctly and not authentified

**Review by antho1404 — COMMENTED** [review]
While splitting is not a bad thing to do I don't believe that will solve any issue (other than possible developer experience issue). I'm good with this split if it's necessary but let's make sure to update the documentation in this case and this would actually be a breaking change so I would do it only if necessary

**Comment by ismailToyran** [comment]
Can be reopened later on.

**Pull request closed without merging**